vizzy/templates/ingress.yaml

```diff
@@ -5,6 +5,7 @@ metadata:
   annotations:
     "nginx.ingress.kubernetes.io/force-ssl-redirect": "true"
     "kubernetes.io/ingress.class": nginx
+    "nginx.ingress.kubernetes.io/proxy-body-size": "100m"
```

Comment:
Can you move this up one line so its next to the other `nginx.ingress.kubernetes.io`?